dts/riscv/riscv32-litex-vexriscv.dtsi

```diff
@@ -32,6 +32,16 @@
 		#size-cells = <1>;
 		compatible = "litex,vexriscv";
 		ranges;
+		ctrl0: soc_controller@e0000000 {
+			compatible = "litex,soc-controller";
+			reg = <0xe0000000 0x4
+				0xe0000004 0x4
+				0xe0000008 0x4>;
+			reg-names = "reset",
+				"scratch",
+				"bus_errors";
+			status = "disabled";
```

Comment:
```suggestion
```
It's disabled by default, so we don't have to do this explicitly here.

Reply:
I just followed the convention of the other nodes, which almost all have the `status = "disabled";`.

Reply:
ack'd